core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -299,13 +299,17 @@ private[data] trait EitherTFunctions {
     *  given `E` in `Left` lifted into the specified `Applicative`.
     *
     * {{{
+    * scala> import cats.data.EitherT
+    * scala> import cats.implicits._
+    * scala> import scala.concurrent.Future
+    * scala> import scala.concurrent.ExecutionContext.Implicits.global
     * scala> val userInput = "hello world"
     * scala> EitherT.cond[Future](
-    *   userInput.forall(_.isDigit) && userInput.size == 10,
-    *   userInput,
-    *   s"The input does not look like a phone number")
-    * res0: EitherT[Future,String,String] =
-    *   EitherT(Future(Success(Left(The input hello world does not look like a phone number))))
+    *    | userInput.forall(_.isDigit) && userInput.size == 10,
+    *    | userInput,
+    *    | s"The input does not look like a phone number")
+    * res0: cats.data.EitherT[scala.concurrent.Future,String,String] =
+    *   EitherT(Future(Success(Left(The input does not look like a phone number))))
```

Comment:
I think the new line here is possibly the reason.

This compiled for me : 

```scala
scala> import cats.data.EitherT
scala> import cats.implicits._
scala> import scala.concurrent.Future
scala> import scala.concurrent.ExecutionContext.Implicits.global
scala> val userInput = "hello world"
scala> EitherT.cond[Future](
     |   userInput.forall(_.isDigit) && userInput.size == 10,
     |   userInput,
     |   "The input does not look like a phone number")
res0: EitherT[Future,String,String] = EitherT(Future(Success(Left(The input does not look like a phone number))))
```

Otherwise the generated test was  : 

```scala
include(new org.scalacheck.Properties("cond") {
  import cats.data.EitherT

  import cats.implicits._

  import scala.concurrent.Future

  import scala.concurrent.ExecutionContext.Implicits.global

  val userInput = "hello world"
})
```

Given how easy it seems for an example to not turn out exactly as we expect, I think the chances are pretty high that we currently have other examples/doctests which are not properly tested.